### DISCUSSION TITLE
No captures, or notifications

### DISCUSSION THREAD

**Discussion by mpdaws** [djdd87/SynoAI#122] [Q&A] [answered]
Hello everyone.  I'm really impressed by this project and have wanted to get improvements like this since I first setup SSS.   

I've run through the setup, and it all looked very promising, but unfortunately I get no files in the capture folder and no notifications.   I've turned logging up to informational level and can see that objects are correctly being identified by the AI, but that's where it stops.  

![image](https://user-images.githubusercontent.com/102721149/160998658-88a3ba8d-809b-46f6-9a76-3b1471231c9c.png)

Any clues on what I might be getting wrong here?  I've read through the readme a few times to see if I've missed something obvious, but nothing is jumping out at me.

Also, less importantly, the TZ variable doesn't seem to be applying (I'm Australia/Brisbane) and everything is displaying GMT.

**Comment by djdd87** [comment]
The TZ issue is something discussed many times in the issue tracker. It's a Synology Docker issue, not a SynoAI issue. The TZ variable only applies to the container, not the logging.

You'll need to provide your config and a screenshot of your volumes directory if you want any help.

**Reply by mpdaws** [reply]
I'll dig into the issues section shortly, thank you.   Other details here.

[appsettings.txt](https://github.com/djdd87/SynoAI/files/8387296/appsettings.txt)

![image](https://user-images.githubusercontent.com/102721149/161008690-2205723b-9dfa-48f0-8f82-b21dbcdc5884.png)

**Comment by djdd87** [comment, marked as answer]
Right, the reason is your min and maxes:

```json
"Cameras": [
{
    "Name": "FrontYard",
    "Types": [ "Person", "Car", "Truck" ],
    "Threshold": 45,
    "MinSizeX": 250,
    "MinSizeY": 500
}    
```

As you'll see in the logs you provided, the size of the detected item is 96x206, which is below the minimum sizes you've defined (96 is less than 250 and 206 is less than 500). The % is OK as you've defined 45% as the threshold and the AI is 46% certain it's a person. So while the AI is confident enough it's a person to meet your threshold, you min and max sizes exclude it.

If you set your logging to DEBUG, then the logs will explain why things were excluded.

**Reply by mpdaws** [reply]
Thank you!   I'm pleased it was nothing more complex.   Hopefully I'll have the remaining setup completed tomorrow.
